config/DoctrineClassMapping/WMDE.Fundraising.AddressChange.Domain.Model.AddressChange.dcm.xml

```diff
@@ -32,6 +32,16 @@
         <option name="fixed"/>
       </options>
     </field>
+    <field name="createdAt" type="datetime" column="created_at" nullable="false">
+      <options>
+        <option name="fixed"/>
+      </options>
+    </field>
+    <field name="modifiedAt" type="datetime" column="modified_at" nullable="false">
+      <options>
+        <option name="fixed"/>
```

Comment:
`<option name="fixed"/>` does not really make sense for these fields and should probably be removed. From the Doctrine docs:

_**fixed**: Boolean value to determine if the specified length of a string column should be fixed or varying (applies only for string/binary column and might not be supported by all vendors). You are also missing `export_date`._

Reply:
I am also wondering if the doc comment mapping in FundraisingStore is missing the alternative underscore naming for modified_at and created_at, it's not mentioned here:

https://github.com/wmde/FundraisingStore/blob/15c54a3ee64ac2b562edcbe65faa68411558efef/src/Entities/AddressChange.php#L92

However, it's used in the database migration script.